### ISSUE TITLE
Migrate snapshot test for alert

### ISSUE THREAD

**Issue by oliverschuerch** [swisspost/design-system#1680]
```[tasklist]
### Tasks
- [x] blocked by #1667
- [x] copy & paste packages/documentation/cypress/snapshots/styles/alert.snapshot.ts to packages/documentation-v7/cypress/snapshots/components/alert.snapshot.ts
- [x] update url to visit
```

**Comment by imagoiq** [comment]
New path for snapshot is defined with this attribute: https://github.com/swisspost/design-system/pull/1671/files#diff-3a3fd3d313363f78d1f85ebf5d3b9a7b5c726fcd814822b7aed0d88b2cf7a8adR8

**Comment by davidritter-dotcom** [comment]
Already done with another issue.
